sdk/src/main/scala/com/horizen/certificatesubmitter/CertificateSubmitter.scala

```diff
@@ -117,7 +118,8 @@ class CertificateSubmitter[T <: CertificateData](settings: SidechainSettings,
       getSignaturesStatus orElse
       submitterStatus orElse
       signerStatus orElse
-      reportStrangeInput
+      reportStrangeInput orElse
+      keyRotationMessageToSign
```

Comment:
just a note for the future `reportStrangeInput`  must be the last case-processor in the chain